docs/release/releases/release-1.7.md

```diff
@@ -0,0 +1,36 @@
+# Cluster API v1.7
+
+## Timeline
+
+The following table shows the preliminary dates for the `v1.7` release cycle.
+
+| **What**                                             | **Who**      | **When**                    | **Week** |
+|------------------------------------------------------|--------------|-----------------------------|----------|
+| Start of Release Cycle                               | Release Lead | Monday 4th December 2023    | week 1   |
+| Schedule finalized                                   | Release Lead | Friday 8th December 2023    | week 1   |
+| Team finalized                                       | Release Lead | Friday 8th December 2023    | week 1   |
+| *v1.5.x & v1.6.x released*                           | Release Lead | Tuesday 2nd January 2024    | week 5   |
```

Comment:
Might want to delay this week to allow for holidays - fine to keep the rest of the timeline the same though

Reply:
@killianmuldoon thanks for review, was not sure initially to delay or cut the first v1.5.x&v1.6.x patch releases week earlier. Delayed it now for a week as suggested: https://github.com/kubernetes-sigs/cluster-api/pull/9635/commits/3a2a0afafaa8523c4528da910df9c8afd642658f